### ISSUE TITLE
Random music playback

### ISSUE THREAD

**Issue by Lartza** [azlux/botamusique#17]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Add a feature to allow the bot to automatically play music on it's own from stuff that it has previously downloaded and played. Toggleable to on and off and only play a random track when the playlist is empty.

Additionally a command to add a selection of random tracks to the playlist could be possible

**Describe alternatives you've considered**
None

**Additional context**
I am pretty confident this requires #15 or something similar to work fully and well

**Comment by Canule** [comment]
While I switched over to mumble,  I have wrote some replay stuff for a Red Discord bot audio module I had running at my own instance,
It never grown in to the repo since the audio module in general was in need for rewrite(which finally seems to be happened).
It saves all request per day(general mood if you will), with some basic management to organize (label/merge/auto clean/delete).

That audio module it self GNU GPL v3, but feel free to review/use/pitfall insight my code chunks for this bot.
https://github.com/Cog-Creators/Red-DiscordBot/pull/366/commits/323f600358e6123a55d169e2faaea8601691a482
